.github/workflows/build.yaml

```diff
@@ -0,0 +1,65 @@
+# https://gist.github.com/farr/26d109d947a7ee4a35e37f7d77d91615
+name: latex-build
+on: [pull_request]
+jobs:
+  build-latex:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Get and print env
+      # https://stackoverflow.com/questions/59077079/how-to-get-pull-request-number-within-github-actions-workflow
+      # https://docs.github.com/en/developers/webhooks-and-events/events/github-event-types#pullrequestevent
+      env:
+          PR: ${{ github.event.pull_request.number }}
+          EVENT_NUMBER: ${{ github.event.number }}
+          URL: ${{ github.api_url }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          OWNER: ${{ github.repository_owner }}
+          REPO: ${{ github.repository }}
+          RUN_URL: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}
+      run: |
+        echo ${{ env.PR }}
+        echo ${{ env.EVENT_NUMBER }}
+        echo ${{ env.URL }}
+        echo ${{ env.OWNER }}
+        echo ${{ env.REPO }}
+        echo ${{ env.RUN_URL }}
+    - name: Install TeXlive
+      run: sudo apt-get install texlive texlive-publishers texlive-science latexmk
+    - name: LaTeX Compile
+      run: pdflatex main.tex
+      run: pdflatex main.tex
```

Comment:
could you delete the duplicated line?

Reply:
Need to run it twice to ensure references and internal links are in final pdf.

Reply:
Bkm is right.

Reply:
> The workflow is not valid. .github/workflows/build.yaml (Line: 32, Col: 7): 'run' is already defined

https://github.com/sfc-aqua/Overview-of-Quantum-Communications-E/actions/runs/4540857355